superchain/superchain.go

```diff
@@ -479,6 +479,13 @@ func canonicalizeSemver(version string) string {
 	return version
 }
 
+type ContractBytecodeImmutables struct {
+	AnchorStateRegistry string `toml:"anchor_state_registry,omitempty"`
+	DelayedWETH         string `toml:"delayed_weth,omitempty"`
+	FaultDisputeGame    string `toml:"fault_dispute_game,omitempty"`
+	MIPS                string `toml:"mips,omitempty"`
+}
+
```

Comment:
Nit: can we move this type to `validation/standard/init.go`, and also the `L1ContractBytecodeHashes` type?

Reply:
done